core/common/src/main/java/alluxio/wire/ThriftUtils.java

```diff
@@ -220,5 +247,30 @@ public static WorkerNetAddress fromThrift(alluxio.thrift.WorkerNetAddress worker
   public static alluxio.thrift.WorkerNetAddress toThrift(WorkerNetAddress workerNetAddress) {
     return workerNetAddress.toThrift();
   }
+
+  /**
+   * Converts wire type to thrift type.
+   *
+   * @param ttlAction {@link TtlAction}
+   * @return {@link TTtlAction} equivalent
+   */
+  public static TTtlAction toThrift(TtlAction ttlAction) {
```

Comment:
that might require converting TtlAction from enum to a class making the actual action a field in the class, but I would prefer to do that for consistency with other classes